libvirt/resource_libvirt_domain.go

```diff
@@ -256,6 +256,9 @@ func resourceLibvirtDomainCreate(d *schema.ResourceData, meta interface{}) error
 		domainDef.Devices.Console = consoles
 	}
 
+	controller := defController{Type:"scsi", Model:"virtio-scsi"}
+	domainDef.Devices.Controller = append(domainDef.Devices.Controller, controller)
```

Comment:
I'll change the code so that the scsi controller is only defined if a scsi disk has been specified.